src/Cachet.php

```diff
@@ -57,7 +57,7 @@ public static function path(): string
      */
     public static function dashboardPath(): string
     {
-        return '/'.ltrim(config('cachet.dashboard_path', app()->joinPaths(rtrim(static::path(), '/'), 'dashboard')), '/');
+        return '/'.ltrim(config('cachet.dashboard_path', app()->joinPaths(rtrim(static::path(), '/\\'), 'dashboard')), '/\\');
```

Comment:
This may be better?

```suggestion
        return '/'.ltrim(config('cachet.dashboard_path', app()->joinPaths(rtrim(static::path(), DIRECTORY_SEPARATOR), 'dashboard')), '/\\');
```